server/paymentProviders/opencollective/collective.js

```diff
@@ -99,12 +99,13 @@ paymentMethodProvider.processOrder = async (order, options = {}) => {
       }
     });
     if (!fromCollectiveHostPaymentMethod) {
-      throw new Error(`Host ${fromCollectiveHost.name} needs to add a credit card to send money to different Hosts`);
+      throw new Error(`Host ${fromCollectiveHost.name} needs to add a credit card to send money`,
```

Comment:
unlike `console.log`, you can't append strings by adding more parameters to `new Error()`. Also, usually, when using ``, we concatenate everything in one string.

Reply:
minor thing, did this because the string was being break along with breaking the line and eslint rule(`[eslint] Unexpected string concatenation. (prefer-template)`) doesn't allow me to use quotes. anyways, will fix it. thanks.

Reply:
don't like it, but will do this(to keep less than 120 character columns per line):
https://stackoverflow.com/questions/38000659/template-strings-es6-prevent-line-breaks